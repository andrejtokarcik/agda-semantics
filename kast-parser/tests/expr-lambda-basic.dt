expr = Lambda x (Set 2) x
