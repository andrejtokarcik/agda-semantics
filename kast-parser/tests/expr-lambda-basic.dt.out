<T>
    <k>
        .K
    </k>
    <mgu>
        subst(.KList)
    </mgu>
    <tenv>
        expr |-> 'Pi___(x,, Set 2,, Set 2)
    </tenv>
    <data>
        .Map
    </data>
    <fun>
        expr |-> lambda ( closure ( x , x , .Map ) )
    </fun>
    <meta>
        .Map
    </meta>
</T>
